bin/genome-env

```diff
@@ -89,7 +102,14 @@ function main {
 
     if test -z "$DISABLE_DB"
     then
-        log.info "Test DB Snapshot: $DB_SNAPSHOT_NAME"
+        if test -n "$DB_ID"
+        then
+            log.info "Using existing test DB"
+            log.info "Test DB Host: ${TESTDBSERVER_DB_HOST}"
+            log.info "Test DB Port: ${TESTDBSERVER_DB_PORT}"
+        else
+            log.info "Test DB Snapshot: $DB_SNAPSHOT_NAME"
+        fi
         log.info "Test DB Name: $TESTDBSERVER_DB_NAME"
```

Comment:
I'm not sure I know what you mean, but if you mean the block starting at line 80, I guess the reason is to maintain the order of the log messages.